### ISSUE TITLE
Can you publish your npm package on npm?

### ISSUE THREAD

**Issue by magikid** [PUGX/PUGXAutoCompleterBundle#72]
I'm working on upgrading this library to v1.7 and running into an issue generating assets.  I've discovered in upgrading this package that it is adding a new line to my package.json: 
```
"@pugx/autocompleter-bundle": "file:vendor/pugx/autocompleter-bundle/assets",
```

The problem I'm having here is that I generate my assets in a previous stage and container from where the PHP code runs so at the time `npm install` is run, not only does this bundle not exist, neither does `composer` for me to install this bundle.  Here is an excerpt of  what I mean:

```
FROM node:8.4.0-alpine as assets-builder
COPY /package.json /package-lock.json /application/
RUN npm install && npm run dump-assets

FROM php:7.4.16-fpm-alpine3.13 as prod
COPY composer.json composer.lock /application/
RUN composer install
COPY --from assets-builder /application/public /application/public
COPY . /application
```

I think this would all be resolved though if you could publish the npm package as npm could grab these assets through npm instead of the file system.  Is that something that you're willing to do?

**Comment by garak** [comment]
The behaviour you're experiencing doesn't depend on this bundle, symfony/flex it's responsible for it.

**Comment by garak** [comment]
It should be fixed in flex 1.13
